tests/security/selinux/semanage_boolean.pm

```diff
@@ -0,0 +1,81 @@
+# Copyright (C) 2020 SUSE LLC
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, see <http://www.gnu.org/licenses/>.
+#
+# Summary: Test "#semanage boolean" command with options
+#          "-l / -D / -m / -C..." can work
+# Maintainer: llzhao <llzhao@suse.com>
+# Tags: poo#64728, tc#1741288
+
+use base 'opensusebasetest';
+use power_action_utils "power_action";
+use strict;
+use warnings;
+use testapi;
+use utils;
+
+sub run {
+    my ($self) = @_;
+    my $test_boolean = "fips_mode";
+
+    $self->select_serial_terminal;
+
+    # list and verify some (not all as it changes often) boolean(s)
+    validate_script_output(
+        "semanage boolean -l",
```

Comment:
1. Does it mean we just compare  the item with prefix(authlogin_ , daemons_ , domain_ , selinuxuser_) in the system for testing?  http://openqa.nue.suse.com/tests/4034257#step/semanage_boolean/9

2. No need to change the content as follows, right?
          authlogin_.*(off.*,.*off).*
            daemons_.*(off.*,.*off).*
            domain_.*(off.*,.*off).*
            selinuxuser_.*(on.*,.*on).*

Reply:
1. the booleans and their default settings will changes very frequently, here just do some ambiguous checking.
fyi:
    validate_script_output(
        "semanage boolean -l",
        sub {
            m/
            authlogin_.*(off.*,.*off).*
            daemons_.*(off.*,.*off).*
            domain_.*(off.*,.*off).*
            selinuxuser_.*(on.*,.*on).*/sx
        }); 

2. sorry, I am not very clear what you mean :(

Reply:
1. For example, I see the content, from the verification run. Do they need to check the "on" status too?

```
domain_can_mmap_files          (on   ,   on)  Allow domain to can mmap files
domain_can_write_kmsg          (off  ,  off)  Allow domain to can write kmsg
domain_fd_use                  (on   ,   on)  Allow domain to fd use
domain_kernel_load_modules     (off  ,  off)  Allow domain to kernel load modules
```


2. Just the same as above, because I see some "on" status, will we need to add them in the checking too?

Reply:
for 2. there is one boolean is "on ", see "selinuxuser_.*(on.*,.*on)"

Reply:
OKay I see, thanks for the detail explanation. LGTM.